contact_links/data/server_action.xml

```diff
@@ -1,11 +1,19 @@
 <odoo>
-	<record model="ir.actions.server" id="action_sync_link">
-        <field name="name">Update Link</field>
-        <field name="model_id" ref="base.model_res_partner"/>
+<!--	<record model="ir.actions.server" id="action_sync_link">-->
```

Comment:
Commented code, write a reasoning why it should be kept otherwise remove it.

Reply:
Ok @nils-vertel I wil add comment. This file was not loaded in manifest file.It means they are not using this server action. I loaded this file in manifest to add new server action that's why commented.

Reply:
Added reason @nils-vertel